lib/chromedriver.js

```diff
@@ -193,10 +194,8 @@ class Chromedriver extends events.EventEmitter {
       const coercedVersion = semver.coerce(version, true);
       if (coercedVersion) {
         // before 2019-03-06 versions were of the form major.minor
-        // so try to see if it fits the mapping
-        const majMinVersion = `${coercedVersion.major}.${coercedVersion.minor}`;
-        if (mapping[majMinVersion]) {
-          version = majMinVersion;
+        if (coercedVersion.major < NEW_CD_VERSION_FORMAT_MAJOR_VERSION) {
```

Comment:
this `if` can be combined now with the above one using &&

Reply:
I think we should throw an error there. If we fail to get the coercedVersion, we are in trouble. It is failing silently right now